packages/web-react/src/components/Accordion/AccordionContext.ts

```diff
@@ -0,0 +1,14 @@
+import { createContext, useContext } from 'react';
+import { AccordionContextProps } from '../../types';
+
+const defaultContext: AccordionContextProps = {
+  open: [],
```

Comment:
Wouldn't it be better to use `null` or `undefined` as default as we do not know in the beginning if this will be a string or an array?

Reply:
fixed